topostats/classes.py

```diff
@@ -46,6 +48,10 @@ class GrainCrop:
         Thresholds used to find the grain.
     filename : str
         Filename of the image from which the crop was taken.
+    threshold : str
```

Comment:
I realise this is required to fit into the structure of the current output but I think we need to think carefully about `threshold` since we have removed the notion of "direction" by removing the `ImageGrainCrop` and `GrainCropDirection`.

Configuration allows a number of thresholds to be set, both could be positive, so what does being "`above`" a threshold actually mean?